libcxx/include/__functional/not_fn.h

```diff
@@ -48,6 +50,27 @@ _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX20 auto not_fn(_Fn&& __f) {
 
 #endif // _LIBCPP_STD_VER >= 17
 
+#if _LIBCPP_STD_VER >= 26
+
+template <auto _Fn>
+struct __nttp_not_fn_t {
+  template <class... _Args>
+  _LIBCPP_NODISCARD_EXT _LIBCPP_HIDE_FROM_ABI constexpr auto operator()(_Args&&... __args) const
```

Comment:
```suggestion
  _LIBCPP_NODISCARD_EXT _LIBCPP_HIDE_FROM_ABI static constexpr auto operator()(_Args&&... __args)
```

I think we can make this operator `static`?

Reply:
`volatile` values can be rejected by not using `static`, but this doesn't seem worthwhile... See [[func.require]/4](https://eel.is/c++draft/func.require#4) and [LWG4007](https://cplusplus.github.io/LWG/issue4007).

Reply:
Initially, I've made `operator()` static but due to the reason previously mentioned

> volatile values can be rejected by not using static

I've made it member function again. For now, I'm going to leave it as-is and wait for maintainers' comment.